exaudfclient/base/javacontainer/script_options/extractor.h

```diff
@@ -3,17 +3,20 @@
 
 #include <string>
 #include <vector>
-#include <set>
+#include <functional>
 
 
 namespace SWIGVMContainers {
 
 namespace JavaScriptOptions {
 
 struct Extractor {
```

Comment:
Does it make sense to add some comment, what the purpose of the converter is

Reply:
Done